spec/grape/api_spec.rb

```diff
@@ -885,6 +885,40 @@ class DummyFormatClass
     end
   end
 
+  describe '.compile!' do
+    it 'requires the grape/eager_load file' do
+      expect(app).to receive(:require).with('grape/eager_load') { nil }
+      app.compile!
+    end
+
+    it 'compiles the instance for rack!' do
+      stubbed_object = double(:instance_for_rack)
+      allow(app).to receive(:instance_for_rack) { stubbed_object }
```

Comment:
@myxoh I am curious about the purpose of this test case. It seems to test nothing?